src/main_dpdk.cpp

```diff
@@ -1906,7 +1959,8 @@ HOT_FUNC int CCoreEthIFStateless::send_node_flow_stat(rte_mbuf *m, CGenNodeState
     lp_s->add_bytes(mi->pkt_len + 4); // We add 4 because of ethernet CRC
 
     if (hw_id >= MAX_FLOW_STATS) {
-        fsp_head->time_stamp = os_get_hr_tick_64();
+        fsp_head->time_stamp = CGlobalInfo::m_options.m_get_latency_timestamp();
```

Comment:
Could be wrapped under different name why it is in ```CGlobalInfo::m_options.```
CGlobalInfo::m_options.m_get_latency_timestamp();